packages/lit-dev-content/site/docs/v3/data/task.md

```diff
@@ -442,3 +442,41 @@ class MyElement extends LitElement {
 ```
 
 {% endswitchable-sample %}
+
+### TypeScript Tips
+
```

Comment:
I'd add an introduction sentence:

```suggestion
Task argument types can sometimes be inferred too loosely by TypeScript. This can be fixed by casting argument arrays with `as const`.
```